app/src/hnqis/java/org/eyeseetea/malariacare/LoginActivityStrategy.java

```diff
@@ -49,12 +50,15 @@ public LoginActivityStrategy(LoginActivity loginActivity) {
 
     public void onCreate() {
         if (existsLoggedUser()) {
-            LoadCredentialsUseCase loadUserAndCredentialsUseCase =
-                    new LoadCredentialsUseCase(loginActivity);
+            LoadCredentialsUseCase loadUserAndCredentialsUseCase =new AuthenticationFactory().getloadCredentialsUseCase(loginActivity);
```

Comment:
It's necessary to invoke load credentials use case here? credentials are not used on this code.